components/purchase_modal/purchase_modal.tsx

```diff
@@ -573,6 +669,9 @@ export default class PurchaseModal extends React.PureComponent<Props, State> {
                                         }}
                                         contactSupportLink={this.props.contactSalesLink}
                                         selectedProduct={this.state.selectedProduct}
+                                        teamName={this.props.team?.name}
+                                        isProratedPayment={(!this.props.isFreeTrial && this.state.currentProduct?.name.toLowerCase().includes('starter')) &&
+                                        !this.state.selectedProduct?.name.toLowerCase().includes('starter')}
```

Comment:
Same as above here, the condition to be prorated is going from a flat-fee to a user-based product. We can generalize this so we don't hardcode any product name

Reply:
ok